apps/prairielearn/src/pages/home/home.html.ts

```diff
@@ -0,0 +1,258 @@
+import { z } from 'zod';
+
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { config } from '../../lib/config.js';
+import { CourseInstanceSchema, CourseSchema } from '../../lib/db-types.js';
+
+export const InstructorCourseSchema = z.object({
+  id: CourseSchema.shape.id,
+  label: z.string(),
+  do_link: z.boolean(),
```

Comment:
"do" is an idiosyncratic prefix here IMO. Maybe `should_link` instead?

Reply:
Replaced with `can_open_course` (semantics instead of implementation) in 05723bf56.